sphinx-docs/screenshot.py

```diff
@@ -23,19 +23,41 @@ def _parse_focus(arg_str):
         return {'id': split_str[0], 'annotation': split_str[1]}
 
 def _parse_command(command):
-    # Return a dictionary with
-    #   action: the action type (if it's recognized)
-    #   options: a list of options
-    raise NotImplementedError()
+    """" Parses a command into action and options.
+
+    Returns a dictionary with following keys:
+       action (string): the action type (if it's recognized)
+       options (list): a list of options
+
+       Both keys have empty values if action type is not recognized.
+    """
+    # Initalize action and opttions with empty values.
+    action = ''
+    options = []
+
+    command_args = command.split(' ')
+    if command_args[0] in ('click', 'send_keys', 'submit'):
```

Comment:
We should do some basic validation, too. Namely, since `click` and `submit` both don't have valid options, we should raise an error or a warning if some are included.

Another thing just occurred to me -- we should also include the `selector` keyword in the returned dictionary.